app/src/main/java/com/paypal/psix/SetupEventFragment.java

```diff
@@ -14,6 +18,40 @@
 
     @Override
     public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.fragment_setup_event, container);
+        View view = inflater.inflate(R.layout.fragment_setup_event, container);
+        addListeners(view);
+
+        return view;
+    }
+
+    private void addListeners(View rootView) {
```

Comment:
Convert this method to use Butterknife (merge from my branch which already has it or just add it to gradle if you don't want to merge).

Obtain the views using inject and add the listeners to them with butter knife as well. cleans up everything!